src/components/Nav/components/ButtonOptions.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import styled, { css } from 'styled-components';
+import { ChevronRight } from 'react-feather';
+
+function ButtonOptions({
+  handleOptionClick,
+  options,
+  link,
+  handleCursorLeave,
+}) {
+  const [loading, setLoading] = useState(true);
+
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      setLoading(false);
+    }, 700);
+
+    return () => clearTimeout(timer);
+  }, []);
+
+  const handleButtonClick = (value, pageId = null) => {
+    if (link === true) {
```

Comment:
ChatService.js 에 86번째줄 보면, 

link는 무조건 true로 들어와서 
```suggestion
    if (link) {
```
이렇게 true랑 비교하지않아도 true가 반활될꺼에요